### DISCUSSION TITLE
Header.html does not consider a CSS class named "title"

### DISCUSSION THREAD

**Discussion by striveandsurvive** [gotenberg/gotenberg#920] [Q&A]
Hi,

During HTML to PDF generation, my header.html was referring to CSS file, style.css. One of the components in Header was not visible on the generated PDF and I had changed many things for that. Finally, I found that the element had a CSS class named "title" and hence it was displayed - while generating the PDF, Gotenmerg would not have liked that class name! I changed the class name, and it worked!!

Is it the expected behavior? Is that class name used anywhere in Gotenberg? I could not find it in any documents.

The problem is solved, but I am curious in knowing the 'why' behind it :)

**Comment by gulien** [comment]
Hello @striveandsurvive 👋 

I'm not sure to understand your issue, but:

* As stated in the documentation, assets are not loaded (i.e., CSS files, scripts, fonts, etc.) in both the `header.html` and `footer.html`.
* The class `title` is used by Chromium to inject the document title (e.g., `<span class="title"></span>`).
